README.md

```diff
@@ -24,3 +24,16 @@ Install [node.js](https://nodejs.org/en/download/).
 npm install
 hugo --gc --minify
 ```
+
+
+## Plugins Generation CI
+
+Plugins generation goes through Travis CI pipeline using [separate brunch](https://github.com/cdapio/website/tree/master-build-plugins).
```

Comment:
typo - branch.

Reply:
Also, let's change it to ... using a dedicated branch.

Reply:
done